neard/src/main.rs

```diff
@@ -48,5 +48,5 @@ fn main() {
     openssl_probe::init_ssl_cert_env_vars();
     near_performance_metrics::process::schedule_printing_performance_stats(Duration::from_secs(60));
 
-    NeardCmd::parse_and_run()
+    NeardCmd::parse_and_run().expect("NeardCmd::parse_and_run failed")
```

Comment:
This would be better: 

```rust
fn main() -> anyhow::Result<()> {
    // We use it to automatically search the for root certificates to perform HTTPS calls
    // (sending telemetry and downloading genesis)
    openssl_probe::init_ssl_cert_env_vars();
    near_performance_metrics::process::schedule_printing_performance_stats(Duration::from_secs(60));

    NeardCmd::parse_and_run()
}
```